app/forms.py

```diff
@@ -153,41 +153,54 @@ class EditProfileForm(RegistrationForm):
         label="Baby has been born",
         required=False
     )
+    image = forms.ImageField()
+    sur_name = forms.CharField(
```

Comment:
This is not needed. You can use `user.last_name` field from the default Django user model.